### PR TITLE
Custom aspect ratio for Image component

### PR DESCRIPTION
**Pull request by synk**
By recent update, Image component can accept custom aspect ratio rather than pre-defined keywords.

**Note that this is a breaking change.**

See Also
https://developers.line.biz/en/reference/messaging-api/#f-image

### CODE REVIEW

File: line-bot-model/src/main/java/com/linecorp/bot/model/message/flex/component/Image.java [hunk 1]
```diff
@@ -138,4 +136,37 @@ public Image(
...
+         */
+        public ImageBuilder aspectRatio(ImageAspectRatio aspectRatio) {
+            this.aspectRatio = aspectRatio.getRatio();
+            return this;
+        }
+
+        /**
+         * Specify custom aspect ratio.
+         */
+        public ImageBuilder aspectRatio(String aspectRatio) {
```
Comment (imasahiro) [inline line-bot-model/src/main/java/com/linecorp/bot/model/message/flex/component/Image.java]: How about adding `aspectRatio(float, float)` as well?

Comment (synk) [reply line-bot-model/src/main/java/com/linecorp/bot/model/message/flex/component/Image.java]: @imasahiro added

**Comment by imasahiro** [comment]
I will merge once you fix the checkstyle error.
```
> Task :line-bot-model:checkstyleTest FAILED
[ant:checkstyle] [ERROR] /home/travis/build/line/line-bot-sdk-java/line-bot-model/src/test/java/com/linecorp/bot/model/message/flex/component/ImageAspectRatioFormatTest.java:1: consecutive empty lines [RegexpMultiline]
[ant:checkstyle] [ERROR] /home/travis/build/line/line-bot-sdk-java/line-bot-model/src/test/java/com/linecorp/bot/model/message/flex/component/ImageAspectRatioFormatTest.java:1: missing copyright header [RegexpSingleline]
```

**Comment by imasahiro** [comment]
Thanks @synk

**Pull request merged**